float64[] center_x
float64[] center_y
float64[] center_z

float64 sphere_radius

int64 cube_number


